i18n/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
This change is probably not needed?

Reply:
agree .. probably appeared automatically by the merge process

Reply:
It's still there though. :)

Reply:
removed